tests/ci/check_spec.py

```diff
@@ -46,11 +44,19 @@
 error = False
 spec_found = False
 
+skip_ci_specs = []
+skip_ci_specs_env = os.getenv('SKIP_CI_SPECS')
+if skip_ci_specs_env:
+    skip_ci_specs = skip_ci_specs_env.rstrip().split(' ')
+
 for spec in sys.argv[1:]:
     if not spec.endswith('.spec'):
         continue
     spec_found = True
-    print("\n--> Scanning spec file %s" % spec)
+    print("--> Scanning spec file %s" % spec)
```

Comment:
Would it be better if line 56 is moved after line 59 ?
I.e. print either `skipping` or `scanning`, not both.

Reply:
good idea